sdk/servicebus/Azure.Messaging.ServiceBus/CHANGELOG.md

```diff
@@ -2,11 +2,6 @@
 
 ## 7.0.0-preview.8 (2020-10-06)
 
-### Fixed
-- Fixed URL issue when requesting management tokens.
-- Fixed URLs in documentation to have appropriate absolute links.
-- Fixed clearing all properties when creating `ServiceBusMessage` based on a `ServiceBusReceivedMessage`.
-
 ### Added
```

Comment:
... Why do all the other entries use `Added`?

Reply:
Yeah, I wasn't aware of this. I thought Added was correct.

Reply:
I'm trying to find the link;  I can never remember where they hide it.

Reply:
https://azure.github.io/azure-sdk/policies_releasenotes.html

Reply:
Isn't this for the azure-sdk repo release notes, as opposed to the change log?

Reply:
@kinelski help..

Reply:
From what @czubair has discussed during the doc reviews, I believe we've been asked to align the change logs with the format as well.

Reply:
If we need to change the changelog format, let's make sure to do it before release. For the betas, I don't think it is as necessary to align.

Reply:
These guidelines are, indeed, for the release notes. It would be great to have all packages following the same pattern, though. It would make the `changelogs` => `release notes` step easier, and it's ideal if we want to make this process fully automatic.